### PR TITLE
UI deployment on ODF4.13

### PR DESCRIPTION
**Pull request by OdedViner**

### CODE REVIEW

**Review by DanielOsypenko — DISMISSED** [review]

**Review by am-agrawa — APPROVED** [review]

**Review by dahorak — DISMISSED** [review]

**Comment by shyRozen** [comment]
@OdedViner why it showing all green and ready to merge when no PR validation run was executed?

**Comment by petr-balogh** [comment]
> @OdedViner why it showing all green and ready to merge when no PR validation run was executed?

@shyRozen the PR validation check appears only when the PR validation job is triggered. It's not a hard requirement to merge the PR to have PV validation check done.

File: ocs_ci/ocs/ui/views.py [hunk 1]
```diff
@@ -46,6 +46,7 @@
     ),
     "search_operator_installed": ('input[data-test-id="item-filter"]', By.CSS_SELECTOR),
     "thin_sc": ('a[id="thin-link"]', By.CSS_SELECTOR),
+    "thin-csi_sc": ("//span[text()='(default) | csi.vsphere.vmware.com']", By.XPATH),
```
Comment (petr-balogh) [inline ocs_ci/ocs/ui/views.py:49]: Why we don't use ID locator here?

And:
Don't we need to add it also to add_capacity here:
https://github.com/red-hat-storage/ocs-ci/blob/d97ce8afad7a9c0f8f651d253add35b3f5235720/ocs_ci/ocs/ui/views.py#L749

Comment (OdedViner) [reply ocs_ci/ocs/ui/views.py:49]: the development team changed the locator.
For `add capacity`, we are using the default sc... [without clicking on sc buttom]

**Review by petr-balogh — APPROVED** [review]

**Comment by petr-balogh** [comment]
@OdedViner I see like 5 failed test cases probably related to UI. Not sure if we would like to first fix only deployment and the rest of tests in different PR, but if they are part of acceptance suite, I think we need to fix them anyway to get the content propagated to stable branch.

**Review by DanielOsypenko — DISMISSED** [review]

File: ocs_ci/ocs/ui/base_ui.py [hunk 2]
```diff
@@ -145,12 +149,15 @@ def do_send_keys(self, locator, text, timeout=30):
...
-            element.send_keys(text)
+            try:
+                element = wait.until(
+                    ec.visibility_of_element_located((locator[1], locator[0]))
+                )
+                element.send_keys(text)
+            except TimeoutException as e:
+                self.take_screenshot()
+                logger.error(e)
+                raise TimeoutException
```
Comment (ebenahar) [inline ocs_ci/ocs/ui/base_ui.py:160]: please create a function with this code and call it from all 3 places

**Review by ebenahar — DISMISSED** [review]

**Review by ebenahar — APPROVED** [review]

**Review by DanielOsypenko — APPROVED** [review]

**Comment by petr-balogh** [comment]
Adding WIP to prevent merge before we have whole acceptance suite passed, otherwise stable pipeliene will be blocke to move any content to stable branch.

**Comment by petr-balogh** [comment]
> Job UNSTABLE

Got confirmation from Oded that non of the tests which failed in our of previous executions (ui suite) is part of acceptance suite, so we should be fine with merge.

**Review by petr-balogh — APPROVED** [review]

**Pull request merged**